app/src/main/java/ru/iteco/fmhandroid/dao/NurseStationDao.kt

```diff
@@ -0,0 +1,30 @@
+package ru.iteco.fmhandroid.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import ru.iteco.fmhandroid.dto.FullNurseStation
+import ru.iteco.fmhandroid.dto.NurseStation
+import ru.iteco.fmhandroid.entity.NurseStationEntity
+
+@Dao
+interface NurseStationDao {
+    @Transaction
+    @Query("SELECT * FROM NurseStationEntity")
+    fun getAllNurseStation(): Flow<List<NurseStation>>
```

Comment:
Во множественном числе
```suggestion
    fun getAllNurseStations(): Flow<List<NurseStation>>
```